library/RAVE/src/RAVE/AppManager.py

```diff
@@ -162,7 +164,18 @@ def _update_selected_face(self, ident):
         Args:
             ident (id): Id of the face selected in the web client
         """
-        self._selected_face = ident
+        self._selected_face = ident["targetId"]
```

Comment:
Should rename the variable indent to something like inputs or payload because it's not just the ID anymore